### PR TITLE
Vas 25 implement app navigation bar component

### PR DESCRIPTION
**Pull request by YuliiaOsadchuk**

### CODE REVIEW

File: src/components/navigation-bar/navigation-bar.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+} from './navigation-bar.styles';
+
+const NavigationBar = (): ReactElement => (
+  <BoxNavBar>
+    <nav aria-label='main mailbox folders'>
+      <List>
+        <ListItem>
+          <Button variant='contained' startIcon={<AddOutlinedIcon />}>
+            Add new project
+          </Button>
```
Comment (b-sht-lionwood) [inline src/components/navigation-bar/navigation-bar.tsx]: add button shoud not be menu item, it is separate item in navbar

File: src/components/navigation-bar/navigation-bar.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  AllInboxOutlinedIconNavBar,
+  DateRangeOutlinedIconNavBar,
+  PriorityHighOutlinedIconNavBar,
+  TodayOutlinedIconNavBar,
+  BoxNavBar,
+} from './navigation-bar.styles';
+
+const NavigationBar = (): ReactElement => (
+  <BoxNavBar>
+    <nav aria-label='main mailbox folders'>
```
Comment (b-sht-lionwood) [inline src/components/navigation-bar/navigation-bar.tsx]: what is the point of nav if you use BoxNavBar upper?

File: src/pages/home/home.tsx [hunk 3]
```diff
@@ -1,5 +1,7 @@
 import React, { ReactElement } from 'react';
 
-const Home = (): ReactElement => <h1>Home page</h1>;
+import NavigationBar from '../../components/navigation-bar/navigation-bar';
+
+const Home = (): ReactElement => <NavigationBar />;
```
Comment (b-sht-lionwood) [inline src/pages/home/home.tsx]: Home should be navigation with main content on righ, as in screen
Screenshot 2021-12-21 at 22 29 14

**Review by b-sht-lionwood — CHANGES_REQUESTED** [review]

File: src/components/navigation-bar/navigation-bar.tsx (hunk 2 above)
Comment (YuliiaOsadchuk) [reply src/components/navigation-bar/navigation-bar.tsx]: BoxNavBar is just a styles div

**Pull request merged**